src/main/java/org/ieeervce/gatekeeper/controller/LoginController.java

```diff
@@ -0,0 +1,64 @@
+package org.ieeervce.gatekeeper.controller;
+
+import org.ieeervce.gatekeeper.ItemNotFoundException;
+import org.ieeervce.gatekeeper.dto.UserDTO;
+import org.ieeervce.gatekeeper.entity.Role;
+import org.ieeervce.gatekeeper.entity.Society;
+import org.ieeervce.gatekeeper.entity.User;
+import org.ieeervce.gatekeeper.service.RoleService;
+import org.ieeervce.gatekeeper.service.SocietyService;
+import org.ieeervce.gatekeeper.service.UserService;
+import org.modelmapper.ModelMapper;
+import org.modelmapper.PropertyMap;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/register")
+public class LoginController {
+    private final UserService userService;
+    private final ModelMapper modelMapper;
+    private final SocietyService societyService;
+    private final RoleService roleService;
+    @Autowired
+    public LoginController(UserService userService, ModelMapper modelMapper,SocietyService societyService,RoleService roleService) {
+        this.userService = userService;
+        this.modelMapper = modelMapper;
+        this.societyService = societyService;
+        this.roleService= roleService;
+       this.modelMapper.addMappings(skipReferencedFieldsMap);
+       this.modelMapper.getConfiguration().setAmbiguityIgnored(true);
+    }
+    PropertyMap<UserDTO,User> skipReferencedFieldsMap = new PropertyMap<UserDTO, User>() {
+        @Override
+        protected void configure() {
+            skip().setUserId(null);
+        }
+    };
+
+    @GetMapping
+    String login(){
+        return ("register");
```

Comment:
I assume this text is placeholder. If so,
please place a TODO/FIXME comment so we can easily track it.

```suggestion
        //FIXME This is a placeholder for the actual logic.
        return "register";
```